src/modules/asset-segments/asset-segments.service.ts

```diff
@@ -0,0 +1,67 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Repository } from 'typeorm';
+import { AssetSegmentsEntity, AssetSegmentsDto } from 'lib-typeorm';
+
+
+@Injectable()
+export class AssetSegmentsService {
+  constructor(
+    @InjectRepository(AssetSegmentsEntity)
+    private readonly assetSegmentsRepository: Repository<AssetSegmentsEntity>,
+  ) {}
+
+  async create(assetSegmentsDto: AssetSegmentsDto): Promise<AssetSegmentsEntity> {
+    const assetSegment = this.assetSegmentsRepository.create(assetSegmentsDto);
+    return this.assetSegmentsRepository.save(assetSegment);
+  }
+
+  async findAll(query: any): Promise<AssetSegmentsEntity[]> {
```

Comment:
Always avoid using `any` in typescript. Also, did you get an error on linting or pre-commit hook here? I am asking because if you didn't get any then I need to fix this on your system as well.

Reply:
I didnt get any error